react/javascript/demo-core-sdk/package.json

```diff
@@ -102,5 +105,13 @@
         }
       ]
     ]
+  },
+  "resolutions": {
```

Comment:
Added this to all package.json to simplify the life of a developer who copies one example